src/world/area_kpa/kpa_114/A9E980.c

```diff
@@ -1,3 +1,7 @@
 #include "kpa_114.h"
 
-INCLUDE_ASM(s32, "world/area_kpa/kpa_114/A9E980", func_80240000_A9E980);
+ApiStatus func_80240000_A9E980(Evt* script, s32 isInitialCall) {
+    sprintf((s8* ) &wMapShapeName, "kpa_112_shape");
```

Comment:
The casts and & shouldn't be needed

Reply:
It doesn't seem to match without &

![image](https://user-images.githubusercontent.com/40916869/188466963-8efe28f4-45ed-427d-9e18-2ff980ea0737.png)

Reply:
can you try typing the shape name and the hit name as `char wMapShapeName[];` and `char wMapHitName[];`?